Core/AppRuntime/Source/AppRuntime.cpp

```diff
@@ -1,11 +1,15 @@
 #include "AppRuntime.h"
-
 #include "WorkQueue.h"
 
 namespace Babylon
 {
     AppRuntime::AppRuntime()
-        : m_workQueue{std::make_unique<WorkQueue>([this] { RunPlatformTier(); })}
+        : AppRuntime{DefaultUnhandledExceptionHandler}
+    {
+    }
+
+    AppRuntime::AppRuntime(std::function<void(std::exception_ptr)> unhandledExceptionHandler)
```

Comment:
I kind of think this function should return a bool indicating whether the unhandled exception is being handled, and if not, then let it fall back to the default.

Reply:
I like it. I'll make the change.

Reply:
I ended up not doing this since it made the code a bit more difficult to understand.